<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaAgregarProducto</class>
 <widget class="QMainWindow" name="ventanaAgregarProducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelAgregarProducto">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AGREGAR PRODUCTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCodigo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>codigo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerie">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>serie:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescripcion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>descripcion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrecioContado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>precio contado:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrecioLista">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>precio lista:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStock">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>stock:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCodigo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSerie">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDescripcion">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrecioContado">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrecioLista">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStock">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelCategoria">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>categoria:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCategoria">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
